src/Workspaces/Core/Portable/CodeFixesAndRefactorings/IFixAllProvider.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using FixAllScope = Microsoft.CodeAnalysis.CodeFixes.FixAllScope;
+
+namespace Microsoft.CodeAnalysis.CodeFixesAndRefactorings
+{
+    internal interface IFixAllProvider
+    {
+        IEnumerable<FixAllScope> GetSupportedFixAllScopes();
```

Comment:
don't love this being IEnumerable for an interanl API.  but we can live with it.